BottomNavigation error on back press (application exit)

**Environment**
 - CLI: 7.0.6
 - Cross-platform modules: 7.0.0
 - Android Runtime: 7.0.0
 - iOS Runtime:
 - Plugin(s):

**Describe the bug**
In NativeScript 7 Android, I experience a RuntimeException error when pressing 'back' to exit from a BottomNavigation-based app. BottomNavigation contains Frames as content items and it seems that is actually the problem.
Let me point out the guilty lines of code in order:

**EDITED**
1) https://github.com/NativeScript/NativeScript/blob/master/packages/core/ui/frame/index.android.ts#L1186
2) https://github.com/NativeScript/NativeScript/blob/master/packages/core/ui/frame/index.android.ts#L959

All in all, 'nativeViewProtected' ends up null in call No 2 because of call No 1. If I comment line in call No 2 or vice-versa, error is gone. That is how I concluded this is the cause.
What I can't explain is that this code is identical to the one in NativeScript 6 sources, but this error never occured before.

**To Reproduce**
In an application that uses a BottomNavigation and Frames as its content items, press back to exit.

**Additional Context**
```
System.err: java.lang.RuntimeException: Unable to destroy activity {org.myapp.demo/com.tns.NativeScriptActivity}: com.tns.NativeScriptException: Calling js method onDestroy failed
System.err: TypeError: Cannot read property 'setBackgroundDrawable' of null
System.err:     at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:3714)
System.err:     at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:3732)
System.err:     at android.app.ActivityThread.access$1400(ActivityThread.java:151)
System.err:     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1395)
System.err:     at android.os.Handler.dispatchMessage(Handler.java:110)
System.err:     at android.os.Looper.loop(Looper.java:193)
System.err:     at android.app.ActivityThread.main(ActivityThread.java:5322)
System.err:     at java.lang.reflect.Method.invokeNative(Native Method)
System.err:     at java.lang.reflect.Method.invoke(Method.java:515)
System.err:     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:829)
System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:645)
System.err:     at dalvik.system.NativeStart.main(Native Method)
System.err: Caused by: com.tns.NativeScriptException: Calling js method onDestroy failed
System.err: TypeError: Cannot read property 'setBackgroundDrawable' of null
System.err:     at com.tns.Runtime.callJSMethodNative(Native Method)
System.err:     at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:1302)
System.err:     at com.tns.Runtime.callJSMethodImpl(Runtime.java:1188)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1175)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1153)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1149)
System.err:     at com.tns.NativeScriptActivity.onDestroy(NativeScriptActivity.java:49)
System.err:     at android.app.Activity.performDestroy(Activity.java:5522)
System.err:     at android.app.Instrumentation.callActivityOnDestroy(Instrumentation.java:1119)
System.err:     at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:3701)
System.err:     ... 11 more
System.err: An uncaught Exception occurred on "main" thread.
System.err: Unable to destroy activity {org.myapp.demo/com.tns.NativeScriptActivity}: com.tns.NativeScriptException: Calling js method onDestroy failed
System.err: TypeError: Cannot read property 'setBackgroundDrawable' of null
System.err: StackTrace:
System.err: java.lang.RuntimeException: Unable to destroy activity {org.myapp.demo/com.tns.NativeScriptActivity}: com.tns.NativeScriptException: Calling js method onDestroy failed
System.err: TypeError: Cannot read property 'setBackgroundDrawable' of null
System.err:     at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:3714)
System.err:     at android.app.ActivityThread.handleDestroyActivity(ActivityThread.java:3732)
System.err:     at android.app.ActivityThread.access$1400(ActivityThread.java:151)
System.err:     at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1395)
System.err:     at android.os.Handler.dispatchMessage(Handler.java:110)
System.err:     at android.os.Looper.loop(Looper.java:193)
System.err:     at android.app.ActivityThread.main(ActivityThread.java:5322)
System.err:     at java.lang.reflect.Method.invokeNative(Native Method)
System.err:     at java.lang.reflect.Method.invoke(Method.java:515)
System.err:     at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:829)
System.err:     at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:645)
System.err:     at dalvik.system.NativeStart.main(Native Method)
System.err: Caused by: com.tns.NativeScriptException: Calling js method onDestroy failed
System.err: TypeError: Cannot read property 'setBackgroundDrawable' of null
System.err:     at com.tns.Runtime.callJSMethodNative(Native Method)
System.err:     at com.tns.Runtime.dispatchCallJSMethodNative(Runtime.java:1302)
System.err:     at com.tns.Runtime.callJSMethodImpl(Runtime.java:1188)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1175)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1153)
System.err:     at com.tns.Runtime.callJSMethod(Runtime.java:1149)
System.err:     at com.tns.NativeScriptActivity.onDestroy(NativeScriptActivity.java:49)
System.err:     at android.app.Activity.performDestroy(Activity.java:5522)
System.err:     at android.app.Instrumentation.callActivityOnDestroy(Instrumentation.java:1119)
System.err:     at android.app.ActivityThread.performDestroyActivity(ActivityThread.java:3701)
System.err:     ... 11 more
```

--- a/.github/ISSUE_TEMPLATE/bug_report.md
+++ b/.github/ISSUE_TEMPLATE/bug_report.md
@@ -12,6 +12,7 @@ Provide version numbers for the following components (information can be retriev
  - Cross-platform modules:
  - Android Runtime:
  - iOS Runtime:
+ - XCode Version:
  - Plugin(s):
 
 **Describe the bug**

--- a/packages/core/ui/frame/index.android.ts
+++ b/packages/core/ui/frame/index.android.ts
@@ -954,10 +954,13 @@ class FragmentCallbacksImplementation implements AndroidFragmentCallbacks {
 			const hasRemovingParent = fragment.getRemovingParentFragment();
 
 			if (hasRemovingParent) {
-				const bitmapDrawable = new android.graphics.drawable.BitmapDrawable(application.android.context.getResources(), this.backgroundBitmap);
-				this.frame._originalBackground = this.frame.backgroundColor || new Color('White');
-				this.frame.nativeViewProtected.setBackgroundDrawable(bitmapDrawable);
-				this.backgroundBitmap = null;
+				const nativeFrameView = this.frame.nativeViewProtected;
+				if (nativeFrameView) {
+					const bitmapDrawable = new android.graphics.drawable.BitmapDrawable(application.android.context.getResources(), this.backgroundBitmap);
+					this.frame._originalBackground = this.frame.backgroundColor || new Color('White');
+					nativeFrameView.setBackgroundDrawable(bitmapDrawable);
+					this.backgroundBitmap = null;
+				}
 			}
 		} finally {
 			superFunc.call(fragment);

